### PR TITLE
Ht15 exercise 3 34 chess

### PR DESCRIPTION
**Pull request by mykolamuntian**
The chessboard field is defined by a pair of natural numbers, each of which does not exceed 8: the first number is the number of the vertical (when counting from left to right), the second is the horizontal number (with a bottom-up count). Given natural numbers a, b, c, d, each of which does not exceed 8.
a) On the field (a, b) is a rook. Write the condition under which it threatens the field (c, d).
b) On the field (a, b) there is an elephant. Write the condition under which it threatens the field (c, d).
c) On the field (a, b) is the king. Write down the condition under which it can go one step to the field (c, d).
d) In the field (a, b) is the queen. Write the condition under which it threatens the field (c, d).
e) There is a white pawn on the field (a, b). Write down the condition under which it can go one step to the field (c, d).
f) There is a black pawn in the field (a, b). Write the condition under which it can go one step to the field (c, d).
g) A horse is located on the field (a, b). Write the condition under which it threatens the field (c, d).

### CODE REVIEW

File: src/com/muntian/ht15_Exercise_3_34/Chess.java [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+        }
+        selectChessFigure(coordinates);
+    }
+
+    private static boolean checkOfCorrectInpuDataEnter(int[] coordinates) {
+        if (coordinates[VERTICAL_OF_FIGURE] == coordinates[VERTICAL_OF_TARGET] &&
+                coordinates[HORIZONTAL_OF_FIGURE] == coordinates[HORIZONTAL_OF_TARGET]) {
+            System.out.println("Input error: Identical figure and target coordinates! Try again");
+            return false;
+        } else {
```
Comment (strazhkoe) [inline src/com/muntian/ht15_Exercise_3_34/Chess.java]: else is redundant

File: src/com/muntian/ht15_Exercise_3_34/Chess.java [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+        System.out.println("Horse             7");
+        while (true) {
+            System.out.println("Enter the number of figure from 1 to 7");
+            Scanner input = new Scanner(System.in);
+            if (input.hasNextInt()) {
+                coordinates[FIGURE_NUMBER] = input.nextInt();
+                if (coordinates[FIGURE_NUMBER] > 0 && coordinates[FIGURE_NUMBER] < 8) {
+                    switch (coordinates[FIGURE_NUMBER]) {
+                        case 1:
+                            System.out.println("You have chosen a rook");
```
Comment (strazhkoe) [inline src/com/muntian/ht15_Exercise_3_34/Chess.java]: You should move the phrase "You have chosen" to a constant.

**Review by strazhkoe — CHANGES_REQUESTED** [review]
Good job! Please fix my minor comments

**Comment by mykolamuntian** [comment]
Task was fixed according to the comments

**Review by strazhkoe — APPROVED** [review]

**Pull request merged**